core/model/time_tracking.py

```diff
@@ -0,0 +1,110 @@
+from __future__ import annotations
+
+import datetime
+from typing import TYPE_CHECKING
+
+from sqlalchemy import (
+    Boolean,
+    CheckConstraint,
+    Column,
+    DateTime,
+    ForeignKey,
+    Integer,
+    String,
+    UniqueConstraint,
+)
+from sqlalchemy.orm import Session, relationship
+
+from core.model import Base, get_one_or_create
+
+if TYPE_CHECKING:
+    from sqlalchemy.orm import Mapped
+
+    from core.model import Identifier
+
+
+class IdentifierPlaytimeEntry(Base):
+    __tablename__ = "identifier_playtime_entries"
+
+    id = Column(Integer, autoincrement=True, primary_key=True)
+
+    identifier_id = Column(
+        Integer,
+        ForeignKey("identifiers.id", onupdate="CASCADE", ondelete="CASCADE"),
+        nullable=False,
+    )
+    timestamp: Mapped[datetime.datetime] = Column(
+        DateTime(timezone=True), nullable=False
+    )
+    total_seconds_played = Column(
+        Integer,
+        CheckConstraint(
+            "total_seconds_played <= 60", name="max_total_seconds_played_constraint"
+        ),
+        nullable=False,
+    )
+    tracking_id = Column(String(64), nullable=False)
+    processed = Column(Boolean, default=False)
+
+    identifier: Mapped[Identifier] = relationship("Identifier", uselist=False)
+
+    __table_args__ = (UniqueConstraint("identifier_id", "tracking_id"),)
+
+
+class IdentifierPlaytime(Base):
```

Comment:
Suggestion: Since I think these entries are "identified" by more than the identifier, I think we should rename this table to `playtime_summaries`.

Reply:
Sure